index.html

```diff
@@ -28,7 +28,7 @@
             <li class="nav-text"><a href="#"><i class="fas fa-fire nav-icon"></i>Trending</a></li>
             <li class="nav-text"><a href="#"><i class="fab fa-youtube nav-icon"></i>Subscription</a></li>
         </div>
-        <hr />
+        <hr/ >
```

Comment:
please consider removing these double line ,let it be one line as it apply on the page example